sample/mpp-library/build.gradle.kts

```diff
@@ -41,4 +41,5 @@ dependencies {
 openApiGenerate {
     inputSpec.set(file("src/swagger.json").path)
     generatorName.set("kotlin-ktor-client")
+    additionalProperties.set(mutableMapOf("nonPublicApi" to "true"))
```

Comment:
i think we should set this property to true as default. for those devs that just update version of library in own projects, they should not got public api by default if all was internal later

Reply:
can you do it, or you need help?

Reply:
It depends if you wanna follow the _openapi client gen_ "standard" that has this option `false` by defalut or you prefer to diverge from it to preserver backward-compatibility (and you're saing that). I can have a look, but in these days I'm pretty busy so it'll take longer [-;

Reply:
@Alex009 I've found 10 minutes this evening to look at it. Btw, this library behaves with a negative logic if you are familiar with openapitools client code gen.